### PR TITLE
Finished Data Model and Handling for Email Approval

### LINKED ISSUE #6: Datastructure for Email Approval
Create Datastructur for storage of email adress and Approval status

### LINKED ISSUE #7: Email Approval Logic
Logic, that verifies controll over ETH Adress, sends Email and then up on POST on approval link with the right credentials uploads proof on Blockchain

### PR DESCRIPTION
**Pull request by SpielmannCode**
Finished Data Model and Handling for Email Approval

Closes #6 
Closes #7

### CODE REVIEW

**Comment by SpielmannCode** [comment]
Closes #7

**Comment by SpielmannCode** [comment]
Also Implemented the Chek option for the Email

File: identity-approver/src/test/java/com/idetix/identityapprover/service/EmailIdentityServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    void approveEmailIdentity() {
+    }
+
+    @Test
+    void getEmailIdentityById() {
+    }
+
+    @Test
+    void updateEmailIdentity() {
+    }
```
Comment (mialbu) [inline identity-approver/src/test/java/com/idetix/identityapprover/service/EmailIdentityServiceTest.java:33]: add todo, so we know quickly what still has to be done.

**Comment by SpielmannCode** [comment]
Will fix Issues on other Pull Request

**Pull request closed without merging**